plugins/redfish/fu-self-test.c

```diff
@@ -27,11 +28,47 @@ fu_test_redfish_common_func (void)
 	g_assert_cmpstr (maca, ==, "00:01:02:03:04:05");
 }
 
+static void
+fu_test_redfish_network_mac_addr_func (void)
+{
+	g_autofree gchar *ip_addr = NULL;
+	g_autoptr(GError) error = NULL;
+
```

Comment:
looking at the CI failures, I think if you're going to have a hard requirement on NM, then you're going to need to let the test suite check for NM and skip tests if it's not there

Reply:
Agreed, fixed.